Cargo.toml

```diff
@@ -7,11 +7,18 @@ license = "MIT OR Apache-2.0"
 name = "omnect-cli"
 readme = "README.md"
 repository = "https://github.com/omnect/omnect-cli"
-version = "0.18.3"
+version = "0.19.0"
 
 [dependencies]
 actix-web = "4.4"
 anyhow = "1"
+azure_core = { git = "https://github.com/Azure/azure-sdk-for-rust.git", tag = "v2023-11-03" }
```

Comment:
why not [v2024-01-05](https://github.com/Azure/azure-sdk-for-rust/releases/tag/v2024-01-05)?